nx-X11/lib/xtrans/Xtrans.c

```diff
@@ -1230,7 +1230,7 @@ TRANS(MakeAllCLTSServerListeners) (char *port, int *partial, int *count_ret,
  * may be used by it.
  */
 
-#if (defined(SYSV) && defined(i386) && !defined(__SCO__)) || defined(WIN32) || defined(__sxg__) || defined(__UNIXOS2__)
+#if (defined(SYSV) && defined(__i386__) && !defined(__SCO__)) || defined(WIN32) || defined(__sxg__) || defined(__UNIXOS2__)
 
```

Comment:
@uli42: please add Backported-to-NX-by: for 79104451e3a0bf77ac239341d68e39162f7bb975.